Makefile-os

```diff
@@ -39,7 +39,7 @@ DOCKER_BAKE_ARGS := \
 	--progress $(DOCKER_PROGRESS) \
 	--metadata-file $(DOCKER_METADATA_FILE) \
 
-ifeq ($(DOCKER_PUSH), true)
```

Comment:
This is irrelevant to the main scope but other checks work with `ifneq` which will accept **any** value as true.

DOCKER_PUSH=1
DOCKER_PUSH=true
DOCKER_PUSH=True
DOCKER_PUSH=False

All mean the same thing (we want to push)

DOCKER_PUSH=

OR simply not adding the flag, means don't push